ansible/roles/zuul/files/zuul/layout.yaml

```diff
@@ -15,35 +14,54 @@ pipelines:
           comment: (?i)recheck(( (?:bug|lp)[\s#:]*(\d+))|( no bug))(\s+clean)?\s*$
         - event: comment-added
           comment: (?i)recheck windows$
+  - name: development
+    description: Pipeline used for Windows CI development. It should always work in silent mode.
+    manager: IndependentPipelineManager
+    precedence: low
+    trigger:
+      gerrit:
+        - event: comment-added
+          comment: (?i)recheck windows(?:-| )exp\s*$
 
-jobs:
-  - name: ^.*$
+project-templates:
+  - name: ci-contrail-windows-template
+    production:
+      - '{jobprefix}-production'
+    development:
+      - '{jobprefix}-development'
 
 projects:
   - name: Juniper/contrail-controller
-    check:
-      - ci-contrail-windows
+    template:
+      - name: ci-contrail-windows-template
+        jobprefix: ci-contrail-windows
```

Comment:
If `jobprefix` is always the same, I don't see a point in passing it as a parameter to template.

Reply:
This is actually template (pattern) itself that is changing (production/development). Do we need a template to achieve this effect (maybe we do - I'm just asking)?